ui/src/VoxettaApp.js

```diff
@@ -15,42 +15,84 @@ limitations under the License. */
 
 import {LitElement, html, css} from 'lit-element';
 
-import {CookieService} from './CookieService';
-import {VoxPrompts} from './components/VoxPrompts';
-import {VoxettaRecordButton} from './VoxettaRecordButton';
-import {VoxettaUserForm} from './VoxettaUserForm';
-import {VoxettaUserIcon} from './VoxettaUserIcon';
+import {CookieService} from './utils/CookieService';
+
+import {VoxettaWaveCanvas} from './components/VoxettaWaveCanvas';
+import {VoxettaPrompts} from './components/VoxettaPrompts';
+import {VoxettaRecordButton} from './components/VoxettaRecordButton';
+import {VoxettaSkipButton} from './components/VoxettaSkipButton';
+import {VoxettaUserForm} from './components/VoxettaUserForm';
+import {VoxettaUserIcon} from './components/VoxettaUserIcon';
 
 /**
  * Possible app states.
  */
 const States = {
-  RECORD_PAGE: 'record_page',
+  ACTIVE_RECORD_PAGE: 'active_record_page',
+  UNACTIVE_RECORD_PAGE: 'unactive_record_page',
```

Comment:
nit: INactive?

Reply:
HAHA I thought there was something with the word unactive but I couldn't think of the correct word--thank you, changing everywhere to inactive